introduction.md

```diff
@@ -0,0 +1 @@
+hij wou niet committen omdat mijn github map geen files erbij maakte, dus heb ik deze een beetje gecheesed
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.